fs/read_file_str.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Read file synchronously and output it as a string.
+ *
+ * @param filename File to read
+ * @param encoding Encoding of the file. Default = "utf-8"
```

Comment:
JSDoc supports specifying defaults:

```suggestion
 * @param [encoding="utf-8"] Encoding of the file.
```

However, since `TextDecoder` defaults to ITF-8, should this just pass on `undefined`?

Reply:
JSDoc supports defaults Without the type definition? Thought it was needed due to my linter. My bad.
About the `undefined` value, it may be better to have the default in the jsdoc instead of using implicit default for automatic API documentation. Don't you think?